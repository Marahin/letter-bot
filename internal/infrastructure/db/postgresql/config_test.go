package postgresql

import (
	"testing"

	"github.com/gkampitakis/go-snaps/snaps"
)

func TestDsn(t *testing.T) {
	// when
	output := Dsn()

	// Then
	snaps.MatchSnapshot(t, output)
}
