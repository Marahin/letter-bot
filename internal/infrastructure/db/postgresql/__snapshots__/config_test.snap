
[TestDsn - 1]
host=db port=5432 user=postgres password=postgres dbname=name sslmode=disable
---
